### PR TITLE
Joke API is in working condition

### PR DESCRIPTION
**Pull request by melihozcan1**
Joke API and the unit tests are implemented.

### CODE REVIEW

File: practice-app/api/tests.py [hunk 1]
```diff
@@ -1,3 +1,13 @@
...
+
+class GettingPlacesTestCase(TestCase):
+    def test_get_places(self):
+        c = Client()
+    
+        response = c.get('/api/joke/ani')
+        self.assertEqual(response.status_code, 200)
+
+        response2 = c.get('/api/joke/animal')
+        self.assertEqual(response2.status_code,200)
```
Comment (uskudarli) [inline practice-app/api/tests.py:13]: Can you also test against a joke that is know to exist. Also, need to handle non-existent cases. What should happen for non 200 returning cases? Should be tested.
What does "Places" have to do with "jokes"?

Comment (melihozcan1) [reply practice-app/api/tests.py:13]: I'm updating the code right now. I have a question though. Should I create a new pull request or is there a way to edit this one to cover the new commit?

**Comment by mecolak** [comment]
It seems good. Previous problem seems to be resolved. It may also list possible categories to choose from when the requested category does not exist.

**Review by melihaydogd — COMMENTED** [review]
A basic API to get a joke about a random thing. It can be improved if you return a joke about a specific story's tag.

**Review by mertlkn — APPROVED** [review]
Works fine.

**Review by niyaziulke — APPROVED** [review]
I didn't detect any errors. It's a fun feature.

**Review by omarr09 — APPROVED** [review]
Adding a Joke API is a very good idea. The changes looks well too

**Review by mecolak — APPROVED** [review]

**Comment by kkadirkkalkan** [comment]
It's a good functionality. Test also okay.

**Review by melihaydogd — APPROVED** [review]

**Review by ahmetsenturk — APPROVED** [review]

**Review by kkadirkkalkan — APPROVED** [review]

**Pull request merged**